filesystem_service/src/tests/test_fsmonitor.py

```diff
@@ -0,0 +1,104 @@
+"""
+Module with unittests for the filesystem monitor
+"""
+
+import collections
+import unittest
+
+from filesystem_service import monitor
+
+InotifyEvent = collections.namedtuple('_INOTIFY_EVENT',
+                                      ['wd', 'mask', 'cookie', 'len'])
+
+
+class FileSystemMonitorTest(unittest.TestCase):
```

Comment:
Better use native `Py.Test` style tests.

Reply:
Sometimes it's not convinient. So, leaving as is for now.